metric/go.mod

```diff
@@ -50,5 +50,6 @@ require (
 	github.com/google/go-cmp v0.5.5
 	github.com/stretchr/testify v1.7.0
 	go.opentelemetry.io/otel v0.19.0
+	go.opentelemetry.io/otel/exporters/metric/prometheus v0.19.0
```

Comment:
I don't think we want to introduce this dependency solely for testing.  Is there a way to invoke this code without pulling in an exporter and its 200+ dependencies?

Reply:
Yes, I' m trying to put the test in [here](https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/metric/prometheus/prometheus_test.go), do you think it will work?